### PR TITLE
Implement envelopes repository

### PR DESCRIPTION
**Pull request by rkondratowicz**

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -12,6 +12,12 @@ management:
 spring:
   application:
     name: Blob Router Service
+  datasource:
+    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}${DRAFT_STORE_DB_CONN_OPTIONS:}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml]: DRAFT_STORE_DB_CONN_OPTIONS ?

**Comment by rkondratowicz** [comment]
@nitinprabhu I've added chart changes to this PR so that we can spot if it's correct

File: charts/reform-scan-blob-router/values.yaml [hunk 2]
```diff
@@ -19,6 +19,12 @@ java:
         - crime-storage-account-primary-key
         - flyway-password
   environment:
+    DB_HOST: blob_router-{{ .Values.global.environment }}.postgres.database.azure.com
+    DB_PORT:'5432'
+    DB_NAME: blob_router
+    DB_USER: blob_router@blob_router-{{ .Values.global.environment }}
+    DRAFT_STORE_DB_CONN_OPTIONS: ?sslmode=require
```
Comment (nitinprabhu) [inline charts/reform-scan-blob-router/values.yaml]: draft store ?

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -12,6 +12,12 @@ management:
 spring:
   application:
     name: Blob Router Service
+  datasource:
+    url: jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}${DB_CONN_OPTIONS:}
```
Comment (nitinprabhu) [inline src/main/resources/application.yaml]: do we nee empty value if DB_CONN_OPTIONS is not set. We don't need : ?

File: src/main/resources/application.yaml (hunk 1 above)
Comment (rkondratowicz) [reply src/main/resources/application.yaml]: oops

File: src/main/resources/application.yaml (hunk 3 above)
Comment (rkondratowicz) [reply src/main/resources/application.yaml]: removed

File: charts/reform-scan-blob-router/values.yaml [hunk 4]
```diff
@@ -19,6 +19,12 @@ java:
         - crime-storage-account-primary-key
         - flyway-password
   environment:
+    DB_HOST: blob_router-{{ .Values.global.environment }}.postgres.database.azure.com
+    DB_PORT: '5432'
+    DB_NAME: blob_router
+    DB_USER: blob_router@blob_router-{{ .Values.global.environment }}
+    DB_CONN_OPTIONS: ?sslmode=require
+
```
Comment (aliveni) [inline charts/reform-scan-blob-router/values.yaml]: does this change need to bump the chart version?

Comment (rkondratowicz) [reply charts/reform-scan-blob-router/values.yaml]: oops, I've updated it, but then merged new master with bumped version. Need to bump again

Comment (rkondratowicz) [reply charts/reform-scan-blob-router/values.yaml]: done

**Pull request merged**